test/unit/math/rev/err/check_less_test.cpp

```diff
@@ -0,0 +1,170 @@
+#include <stan/math/rev/arr.hpp>
+#include <stan/math/rev/mat.hpp>
+#include <stan/math/rev/scal.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+#include <limits>
+
+using stan::math::check_less;
+using stan::math::var;
+
+TEST(AgradRevErrorHandlingScalar, CheckLessVarCheckVectorized) {
+  using stan::math::check_less;
+  using stan::math::var;
```

Comment:
Already declared outside of test block.

Reply:
I removed the global using and added it inside separate tests.